examples/e2e/cluster_group/README.md

```diff
@@ -0,0 +1,38 @@
+# Virtual Cluster demo
+
+End-to-end example of provisioning a Palette Virtual Cluster with all of its dependencies. This terraform configuration will provision the following resources on Spectro Cloud:
+- K8s host cluster and datasource.
+- Addon Cluster Profile
+
+## Instructions:
+
+Clone this repository to a local directory, and then change directory to `examples/e2e/virtual`. Proceed with the following:
+1. Provision host cluster as a prerequisite to use virtual cluster. 
+To achieve it add host configuration block to existing cluster or provision it from scratch using one of existing examples:
+<pre>
+  host_config {
+    host_endpoint_type = "LoadBalancer" 
+    ingress_host       = "*.dev.spectrocloud.com"
+  }
+</pre>
+2. From the current directory, copy the template variable file `terraform.template.tfvars` to a new file `terraform.tfvars`.
+3. Specify and update all the placeholder values in the `terraform.tfvars` file.
+4. Initialize and run terraform: `terraform init && terraform apply`.
```

Comment:
```suggestion
4. Initialize Terraform and invoke the deployment with the following command: `terraform init && terraform apply --auto-approve`.
```
@nikchern Not sure if you want us to include the `--auto-approve` flag but otherwise we should mention the prompt.

Reply:
Will include auto approve.